frontend/app/services/EventbriteService.scala

```diff
@@ -207,6 +227,10 @@ object EventbriteService {
     GuardianLiveEventService.getPreviewEvent(id)
   }
 
+  def getPreviewDiscoverEvent(id: String): Future[RichEvent] = Cache.getOrElse[Future[RichEvent]](s"preview-event-$id", 2) {
+    DiscoverEventService.getPreviewEvent(id)
+  }
+
   def getPreviewMasterclass(id: String): Future[RichEvent] = Cache.getOrElse[Future[RichEvent]](s"preview-event-$id", 2) {
```

Comment:
If we have a preview method you'll need to add it to the routes file and Event controller.